playground/PostgresEndToEnd/PostgresEndToEnd.JavaService/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Use Microsoft's OpenJDK image with Maven for building
```

Comment:
Unused, but could be for deployment, at least keeping it for reference as it's working